### PR TITLE
powersuit armour item,powerglove reskins. uhhh ready 2 merge

### PR DESCRIPTION
**Pull request by ghost**
https://i.gyazo.com/bc8102f4cbcc9f505521272aaf5856e8.png

https://i.gyazo.com/9a8643055e610d477ce0e81e2bad4db7.png

risky item,heavy radiation protection and brute/lasor protection but
incredibly weak to electronic attacks
(shocks and SLOWS the clothing down forever)

### CODE REVIEW

**Comment by danhibiki** [comment]
no changelog :-1:
add chagloeg plz

**Comment by ghost** [comment]
oh FUCK I FORGOT TO ADIT

**Comment by ghost** [comment]
addednerd

**Comment by danhibiki** [comment]
Might just be minor nitpicking for no reason on my part, but shouldn't TD be creditted in the changelog too?

**Comment by ghost** [comment]
mmm...gagsa..

**Comment by SplergeBucket** [comment]
so this an admin-only spawn thing or hwat

**Comment by danhibiki** [comment]
Admin spawn only, I suggested potentially putting it off Z and the idea of putting it in the abandoned_crates.dm was suggested as well. I don't think its a problem to have 1 suit laying around.

**Comment by ghost** [comment]
went and put it into abandoned crates aswell as put in some adminspawn only guns for the biker bar

File: code/modules/mining/abandoned_crates.dm [hunk 1]
```diff
@@ -153,6 +153,9 @@
 			new /obj/item/clothing/mask/luchador(src)
 		if(100)
 			new /obj/item/clothing/head/bearpelt(src)
+		if(101 to 110)
```
Comment (Chronitonity) [inline code/modules/mining/abandoned_crates.dm]: Why is this chance much higher than almost everything else?

**Comment by Chronitonity** [comment]
:-1: On the abandoned crate addition, you gave it a much higher chance than actually anything else to appear in those crates. It should have a 1% chance like basically everything else.

**Comment by Carbonhello** [comment]
Spiega temi la nomenclatura

**Comment by Carbonhello** [comment]
Uhh accidentally sent a message here,ignote it

File: html/changelogs/Oprayx73-Powerarmour.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Added t45d power armour, not implemented in ANY way... yet."
+
```
Comment (Carbonhello) [inline html/changelogs/Oprayx73-Powerarmour.yml]: What are those empty lines? They could fuck up the changelog

Comment (ghost) [inline html/changelogs/Oprayx73-Powerarmour.yml]: they dont

Comment (Chronitonity) [inline html/changelogs/Oprayx73-Powerarmour.yml]: Still shouldnt be there to be honest

Comment (ghost) [inline html/changelogs/Oprayx73-Powerarmour.yml]: notepad file is completely fine

Comment (Chronitonity) [inline html/changelogs/Oprayx73-Powerarmour.yml]: You shouldn't be putting empty lines in the changelog at all though, it makes it look ugly iirc

Comment (ghost) [inline html/changelogs/Oprayx73-Powerarmour.yml]: f i x e d

**Comment by Carbonhell** [comment]
conflicts

**Comment by ghost** [comment]
ficksin dis shit

**Comment by nalarmg** [comment]
You don't need to worry about closing your issue if you're fixing it @oprayx73

**Comment by danhibiki** [comment]
Sure

**Comment by trifornt-tinfornt** [comment]
Sure wish you could order a powersuit with traitor gear so you can use a powerglove with it.

**Comment by ghost** [comment]
the coool thing about power ARMOUR is that its gonna be a z level item
that isn't related to traitors or antags at all

**Comment by Lawligagger** [comment]
Sure thing.

**Pull request merged**